h2o-algos/src/main/java/hex/coxph/CoxPH.java

```diff
@@ -727,6 +783,7 @@ public void computeImpl() {
 
     @Override
     protected void chunkInit(){
+      System.out.println("_time = " + Arrays.toString(_time));
```

Comment:
you can also use the profiling feature if that is what you are doing - MRTask.profile()...